components/bl-list-component/component.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "c_f95c74e56529269bdafe747715db7bbe",
+  "name": "List",
+  "description": "The List component allows the user to add a standard list to an application.",
+  "showInToolbox": true,
+  "faIcon": "list",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "select",
```

Comment:
Need to add handler descriptions

Check example https://github.com/Backendless/ui-builder-library/blob/main/components/bl-leafletMap-component/component.json